### PR TITLE
Seed data adam

### PR DESCRIPTION
**Pull request by adamkohler**
Let me know what you all think of this.  For now it is only inputting basic athlete info for 5 athletes into the AthleteUserViewModel since that model had what I needed for the basic search.  We will need to figure out if we add the ApplicationUser to the db as Daniel suggested as an idea to me today.  DB is up and working btw.

### CODE REVIEW

**Comment by Daniel-Watkins** [comment]
With the database now up and running I think we should stay after class tomorrow and take a look at the site as, if memory serves me right, we couldn't do that until the database was created.

**Comment by adamkohler** [comment]
add data for advanced search parameters

**Review by ksdurg — COMMENTED** [review]
looks good to me.

File: gybitg/SeedData.cs [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                return; //already has data, don't add any more test data
+            }
+            if (context.AthleteStats.Any())
+            {
+                return;
+            }
+
+            // Trying to get data for the db context for testing
+            //this was not working correctly
+            //var firstName = new[] { "Adam", "Alex", "Kevin", "Daniel", "Keith" };
```
Comment (Bansenauer-Cascadia) [inline gybitg/SeedData.cs:33]: Take a look at http://jackhiston.com/2017/10/1/how-to-create-bogus-data-in-c/ for some help with this when needed.

**Review by Bansenauer-Cascadia — COMMENTED** [review]
I would suggest you wait till you have the Bogus generator working before merging in seed data for the server. In the mean time, you can use the four athletes that Adam created within a new local Repository

**Review by AlexKSparks — APPROVED** [review]

File: gybitg/SeedData.cs (hunk 1 above)
Comment (ksdurg) [reply gybitg/SeedData.cs:33]: Great post/info.  Thanks, Brian!

File: gybitg/Migrations/ApplicationDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+                    b.Property<string>("UserId")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<string>("AAUId");
+
+                    b.Property<string>("Achievments");
+
+                    b.Property<int>("Lossess");
+
+                    b.Property<string>("PersnalBio");
```
Comment (ksdurg) [inline gybitg/Migrations/ApplicationDbContextModelSnapshot.cs:185]: very small point:  PersnalBio is misspelled.  Should be PersonalBio.  It must have been originally misspelled in the CoachProfile model, that's why Intellisense misspelled it for you.

**Comment by ksdurg** [comment]
Looks good.

File: gybitg/SeedData.cs (hunk 1 above)
Comment (adamkohler) [reply gybitg/SeedData.cs:33]: Yep, my next focus has been the local repo for the four athletes.

**Comment by adamkohler** [comment]
Will need to modify seed data based on meeting with client.  Get with team to finalize the information needed for datasets.

**Pull request closed without merging**